pyairtable/api/table.py

```diff
@@ -25,10 +33,13 @@ def __init__(self, api_key: str, base_id: str, table_name: str, *, timeout=None)
 
         Keyword Args:
             timeout(``Tuple``): |arg_timeout|
+            max_records_per_request: |arg_max_records_per_request|
         """
         self.base_id = base_id
         self.table_name = table_name
-        super().__init__(api_key, timeout=timeout)
+        super().__init__(
+            api_key, max_records_per_request=max_records_per_request, timeout=timeout
+        )
```

Comment:
Carrying through my suggestions for keyword-only arguments...
```suggestion
        super().__init__(api_key, **kwargs)
```